apollo-router/tests/snapshots/set_context__set_context_unrelated_fetch_failure.snap

```diff
@@ -135,7 +135,47 @@ expression: response
                     ]
                   },
                   "path": [
-                    "",
+                    "t",
+                    "u"
+                  ]
+                },
+                {
+                  "kind": "Flatten",
```

Comment:
This Flatten node and the node above are re-ordered in Rust QP. The first Flatten node under the Parallel node should be deleted now. Then, the only changes will be dropping empty path component and schemaAwareHash values.

Reply:
My diff:
```
@@@ -37,55 -37,55 +37,13 @@@ expression: respons
                "operationKind": "query",
                "operationName": "Query_fetch_failure__Subgraph1__0",
                "outputRewrites": null,
--              "schemaAwareHash": "d321568b33e32986df6d30a82443ebb919949617ffc33affe8b413658af52b8a",
++              "schemaAwareHash": "cb2490c26be37beda781fc72a6632c104b5e5c57cfabb5061c7f1429582382d9",
                "serviceName": "Subgraph1",
                "variableUsages": []
              },
              {
                "kind": "Parallel",
                "nodes": [
--                {
--                  "kind": "Flatten",
--                  "node": {
--                    "authorization": {
--                      "is_authenticated": false,
--                      "policies": [],
--                      "scopes": []
--                    },
--                    "contextRewrites": null,
--                    "id": null,
--                    "inputRewrites": null,
--                    "kind": "Fetch",
--                    "operation": "query Query_fetch_failure__Subgraph2__1($representations:[_Any!]!){_entities(representations:$representations){...on U{b}}}",
--                    "operationKind": "query",
--                    "operationName": "Query_fetch_failure__Subgraph2__1",
--                    "outputRewrites": null,
--                    "requires": [
--                      {
--                        "kind": "InlineFragment",
--                        "selections": [
--                          {
--                            "kind": "Field",
--                            "name": "__typename"
--                          },
--                          {
--                            "kind": "Field",
--                            "name": "id"
--                          }
--                        ],
--                        "typeCondition": "U"
--                      }
--                    ],
--                    "schemaAwareHash": "a9aa68bb30f2040298629fc2fe72dc8438ce16bcdfdbe1a16ff088cf61d38719",
--                    "serviceName": "Subgraph2",
--                    "variableUsages": []
--                  },
--                  "path": [
--                    "",
--                    "t",
--                    "u"
--                  ]
--                },
                  {
                    "kind": "Flatten",
                    "node": {
@@@ -128,7 -128,7 +86,7 @@@
                          "typeCondition": "U"
                        }
                      ],
--                    "schemaAwareHash": "da0e31f9990723a68dbd1e1bb164a068342da5561db1a28679693a406429d09a",
++                    "schemaAwareHash": "6679327c6be3db6836531c0d50d583ad66b151113a833f666dcb9ea14af4c807",
                      "serviceName": "Subgraph1",
                      "variableUsages": [
                        "contextualArgument_1_0"
@@@ -171,7 -171,7 +129,7 @@@
                          "typeCondition": "U"
                        }
                      ],
--                    "schemaAwareHash": "a3c7e6df6f9c93b228f16a937b7159ccf1294fec50a92f60ba004dbebbb64b50",
++                    "schemaAwareHash": "cde44282bd7d30b098be84c3f00790e23bbdd635f44454abd7c22b283adba034",
                      "serviceName": "Subgraph2",
                      "variableUsages": []
                    },
```